components/BlogListing.js

```diff
@@ -0,0 +1,67 @@
+const dotCMSApi = require('../utils/dotcms/dotcmsApi');
+
+import Link from 'next/link';
+import DotcmsImage from './DotcmsImage';
+import Loader from './Loader';
+import useDotCMSApi from '../hooks/useDotCMSApi';
+
+const DateText = ({ value }) => {
+    const monthNames = [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December'
+    ];
+    const d = new Date(value);
+
+    return <time dateTime={value}>{`${monthNames[d.getMonth()]} ${d.getUTCDate()}, ${d.getFullYear()}`}</time>;
+};
+
+const BlogListing = () => {
+    const [loading, posts] = useDotCMSApi(async () => {
```

Comment:
I think `async` is not needed

Reply:
It does because I'm returning a `dotCMSApi.esSearch` which is a promise.

Reply:
Good catch.